frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -543,14 +550,22 @@ local function genMetadataFolderMenuItem(value)
     }
 end
 
-common_settings.document_metadata_folder = {
+common_settings.document_metadata_location = {
     text_func = function()
         local value = G_reader_settings:readSetting("document_metadata_folder", "doc")
-        return T(_("Book metadata folder: %1"), metadata_folder_str[value])
+        return T(_("Book metadata location: %1"), metadata_folder_str[value])
     end,
+    help_text = metadata_folder_help_text,
     sub_item_table = {
         genMetadataFolderMenuItem("doc"),
         genMetadataFolderMenuItem("dir"),
+        {
+            text = _("About book metadata location"),
```

Comment:
In all other menus having a "About..." item, this "About..." is above the other items it explains (a bit random decision, but we like consistency ;)

Reply:
Except `Save book metadata`
https://github.com/koreader/koreader/blob/6ad0733f14a47e25ccb38851bdea3849db2ef562/frontend/ui/elements/common_settings_menu_table.lua#L584-L599

Reply:
I must have forgotten about that one :) (possibly one of the first/early "About/more info.." item we added?)
Also, it is conditional (not seeing it on the emulator), so dunno if because of that if feels better at bottom.